src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drowable {
```

Comment:
let's implement `Drawable` in `Figure` and remove `implements Drawable` from classes that `extends Figure`